lib/posix-user/user.h

```diff
@@ -1,8 +1,12 @@
 /* SPDX-License-Identifier: BSD-3-Clause */
 /*
- * Authors: Costin Lupu <costin.lupu@cs.pub.ro>
```

Comment:
Any reason for Costin Lupu to be removed as author? His contributions have been removed from the code?

Reply:
This is actually a new file and I just copied over the header. I removed Felipe and Simon as authors.

Reply:
roger